src/hotspot/share/gc/parallel/psAdaptiveSizePolicy.cpp

```diff
@@ -349,7 +349,7 @@ void PSAdaptiveSizePolicy::compute_eden_space_size(
     log_debug(gc, ergo)(
           "PSAdaptiveSizePolicy::compute_eden_space_size: gc time limit"
           " gc_cost: %f "
-          " GCTimeLimit: " UINTX_FORMAT,
+          " GCTimeLimit: " UINT32_FORMAT,
```

Comment:
We use `%u` as format specifiers for uint.

Reply:
`%u` is used.